scripts/py_matter_idl/matter/idl/tests/outputs/large_all_clusters_app/cpp-app/ember-cluster-config/AccessControl.h

```diff
@@ -0,0 +1,33 @@
+// DO NOT EDIT - Generated file
+//
+// Application configuration for AccessControl based on EMBER configuration
+// from inputs/large_all_clusters_app.matter
+#pragma once
+
+#include <app-common/zap-generated/cluster-enums.h>
+#include <app/util/cluster-config.h>
+
+#include <array>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace AccessControl {
+namespace EmberApplicationConfig {
```

Comment:
So I have to ask: why is "Ember" in the naming here?  This is basically some sort of static generated cluster configuration, or maybe "fixed endpoint" configuration?  It's not necessarily tied to "ember" in any way (though at the moment we are in fact generating it from a ZAP config).

It might be better to just call it directly FixedEndpointConfiguration or something.

Similar for the namespacing of the ClusterConfiguration type and the naming of the constant.

Reply:
This is explicitly ember: this is the static fixed endpoints intended for use in the ember layer, which will be also the one that implements `emberAf...ClusterInitCallback` (the others we did name Matter, but it still ember that calls it).

I am concerned if we ever consider this being outside codegen from ember, hence me adding it in ember directory and ember namespace, to make sure only an ember compatibility layer uses it.

Reply:
> this is the static fixed endpoints intended for use in the ember layer

Where is the "for use in the ember layer" coming from?  This is static fixed endpoint data that could be used by whatever wants to use that, no?  ember is one thing that currently wants such data, but I see no reason some other setup, not tied to the ember frameworking or ZAP in any way, might not want to use it....

Reply:
Offline discussion result:
- will try to keep it generic and place the namespace as `StaticApplicationConfig` (and remove ember from others) ... we do not expect this to be used currently, but it avoids questions of `what is ember`

- a prerequisite PR will be created for "use latest all-clusters for golden image tests"

Reply:
#38326 is the PR for golden image update.